packages/storybook/src/stories/ino-card/ino-card.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { expect, Locator, test } from '@playwright/test';
+import { goToStory } from '../test-utils';
+
+test.describe('ino-card', () => {
+  let inoCard: Locator;
+
+  test.beforeEach(async ({ page }) => {
+    await goToStory(page, ['Structure', 'ino-card', 'default']);
+    inoCard = page.locator('ino-card');
+  });
+
+  test('should render with selected set to true', async () => {
```

Comment:
I think we should not check for classes in ui tests. The test below does suffice IMO.